src/Microsoft.Health.Functions.Extensions/AzureFunctionsJobHostBuilder.cs

```diff
@@ -78,7 +96,14 @@ public IHost Build()
                         .AddEnvironmentVariables();
                 })
             .ConfigureLogging((c, b) => _configureLogger(c, b))
-            .Build();
+            .ConfigureServices(services =>
+            {
```

Comment:
nit: you don't need the curly braces since this can be expressed as a single... expression.